app/jobs/reports/combined_invoice_supplement_report_v2.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Reports
+  class CombinedInvoiceSupplementReportV2 < BaseReport
+    REPORT_NAME = 'combined-invoice-supplement-report-v2'
+
+    def perform(_date)
+      csv = build_csv(IaaReportingHelper.iaas, IaaReportingHelper.partner_accounts)
+      save_report(REPORT_NAME, csv, extension: 'csv')
+    end
+
+    # @param [Array<IaaReportingHelper::IaaConfig>] iaas
+    # @return [String] CSV report
+    def build_csv(iaas, partner_accounts)
```

Comment:
we should update the comment here or remove it
```suggestion
    # @param [Array<IaaReportingHelper::IaaConfig>] iaas
    # @param [Array<IaaReportingHelper::PartnerConfig>] partner_accounts
    # @return [String] CSV report
    def build_csv(iaas, partner_accounts)
```